instance/models.py

```diff
@@ -1,68 +1,317 @@
+from copy import copy
 import datetime
 
 from django.conf import settings
 from django.contrib.auth.models import User
+from django.contrib.contenttypes.fields import GenericRelation
+from django.contrib.contenttypes.models import ContentType
 from django.core import mail
-from django.db import models
+from django.db import models, transaction
+from django.db.models import Prefetch
 from django.db.models.signals import post_save
 from django.utils.translation import ugettext_lazy as _
 
 from annoying.fields import AutoOneToOneField
 from autoslug import AutoSlugField
-from nuntium.popit_api_instance import PopitApiInstance
-from popit.models import Person, ApiInstance
+from popolo.models import (
+    Identifier as PopoloIdentifier,
+    Person,
+)
+from popolo_sources.models import LinkToPopoloSource, PopoloSource
+from popolo_sources.importer import PopoloSourceImporter
 from requests.exceptions import ConnectionError
 from subdomains.utils import reverse
 
+from contactos.models import Contact
+from mailit import MailChannel
+
+
+class PopoloPersonQuerySet(models.QuerySet):
+
+    def get_from_api_uri(self, uri_from_api):
+        try:
```

Comment:
I've made the fix that you suggested in this fixup: d579dabc0e70c5490d712b50823e8e82fdc150d8